color_picker/build/default/intermediates/merge_profile/default/module.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Delete all the generated files like build folder, .preview, .idea, package-lock.json. Handle these changes for all the folders in the project. (Can add these files in .gitignore)

Reply:
Added all the generated files in .gitignore